vnext/ReactWindowsCore/Modules/PlatformConstantsModule.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "PlatformConstantsModule.h"
+#include <VersionHelpers.h>
+
+using Method = facebook::xplat::module::CxxModule::Method;
+
+namespace facebook::react {
+
+const char *PlatformConstantsModule::name = "PlatformConstants";
+
+std::string PlatformConstantsModule::getName() {
+  return PlatformConstantsModule::name;
+}
+
+std::vector<Method> PlatformConstantsModule::getMethods() {
+  return {};
+}
+
+std::map<std::string, folly::dynamic> PlatformConstantsModule::getConstants() {
+  return {
+      // We don't currently treat Native code differently in a test environment
+      {"isTesting", false},
+
+      // Since we're out-of-tree, we don't know the exact version of React Native
+      // we're paired with. Provide something sane for now, and try to provide a
+      // better source of truth later. Tracked by Issue #4073
+      {"reactNativeVersion", folly::dynamic::object("major", 0)("minor", 61)("patch", 0)}
```

Comment:
>0 [](start = 86, length = 1)

0.61.5 would be more accurate

Reply:
I was thinking about that. I have a fear that we would end up not bumping it when doing patch release changes though. I.e. CI guarantees the minor version is right, but we don't have a process to update patch.

At that point zero might be better than having something off, especially if we end up having a loose peer dependency and build something unexpected. I'm going to try to make the upstream change to get real version info soon though.